scripts/sv_spark_pipeline_launch.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+if [[ -z ${1+x} ]]; then
+    echo "please provide address for output directory you have write access to"
+    exit 1
+else
+    OUTPUT=$1
+    echo "output directory is set by user to '$1'"
+fi
+
+# STAGE 1 args
+ONE_MEGA_BYTE=1000000
+TEN_MEGA_BYTES=10000000
+EXCLUSIONINTERVALS=hdfs://svdev-1-m:8020/reference/GRCh37.kill.intervals
```

Comment:
Actually replace "svdev-1" with the cluster name param, and then make another variable with the master node name.